Website/Packages/datax-pipeline/src/modules/flowDefinition/flowModels.js

```diff
@@ -57,6 +58,36 @@ export const sinkerFormatTypeEnum = {
     json: 'json'
 };
 
+export const sinkerSqlWriteModeEnum = {
+    append: 'append',
+    overwrite: 'overwrite',
+    ignore: 'ignore',
+    errorifexists: 'errorIfExists'
+};
+
+export const sinkerSqlWriteModes = [
+    {
+        key: sinkerSqlWriteModeEnum.append,
+        name: 'Append',
+        disabled: false
+    },
+    {
+        key: sinkerSqlWriteModeEnum.overwrite,
+        name: 'Overwrite',
+        disabled: false
+    },
+    {
+        key: sinkerSqlWriteModeEnum.ignore,
+        name: 'Ignore',
+        disabled: false
+    },
+    {
+        key: sinkerSqlWriteModeEnum.errorIfExists,
+        name: 'ErrorIfExists',
```

Comment:
since this is displayed in UI, should you use "Error if exist" as the display name?